README.md

```diff
@@ -99,11 +101,21 @@ Note that Kokkos' runtime library gets built on the fly, and the
 `libkokkos.a` and the intermediate object files are placed to the
 working directory.
 
-### Intel oneAPI
+### SYCL
 
-The beta version of Intel oneAPI can be obtained from https://software.intel.com/en-us/oneapi .
+The test program relies on the In-Order Queues and Unified Shared Memory extensions
+to SYCL 1.2.1, which are currently available in Intel oneAPI toolchain and in the
+LLVM SYCL branch.
 
-The tests programs choose an OpenCL device at runtime using the `cl::sycl::default_selector`.
+The beta version of Intel oneAPI can be obtained from
+https://software.intel.com/en-us/oneapi .
+
+The in-development version of the LLVM compiler with support for SYCL, Intel's
+extensions, and Codeplay's CUDA backend is available on GitHub at
+https://github.com/intel/llvm/ .
+
+The test program should run on any available SYCL device, and can select it at
+runtime based on the command line options.
```

Comment:
Could the list of PRs needed for CUDA backend to be added here as well?

Reply:
I would prefer to set up some instruction on a separate page (like for Alpaka), to avoid making PRs every time they change.

Reply:
Ok, that's fine as well. I'd just want to avoid looking instructions from PRs.